assets/src/edit-story/components/keyboard/index.js

```diff
@@ -0,0 +1,164 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * External dependencies
+ */
+import Mousetrap from 'mousetrap';
+
+/**
+ * WordPress dependencies
+ */
+import { useContext, useEffect, createRef } from '@wordpress/element';
+
+/**
+ * Internal dependencies
+ */
+import Context from './context';
+
+const PROP = '__WEB_STORIES_MT__';
+const INPUT_ELEMENTS = ['INPUT', 'TEXTAREA'];
```

Comment:
I think the naming of this is a little wrong, it is not inputs, as select an others would. I think Editable elements might be better.

Reply:
Yeah, what if focus is on a checkbox or radio, which are also `<input />`s? We'd still want global shortcuts to work, or? What's the proper check to see if an `<input />` is actually an editable input field?

Reply:
We'd probably have to test the `type` attribute against a list.

Reply:
There are also input type button and submit that would not work here.

Reply:
Good points. Done.